server/controllers/finance/reports/budget_analytical/report.handlebars

```diff
@@ -0,0 +1,181 @@
+          </tr>
+        </tbody>
+        <thead>
+          <tr class="text-center" style="background-color: #c6c6c6; text-transform: uppercase; font-size: 14pt;">
+            <th colspan="{{ colspanValue }}" class="text-center"> {{translate 'REPORT.BUDGET_REPORT.EXPENSES'}} </th>
+          </tr>          
+          <tr style="text-transform: uppercase; vertical-align: top;">
+            <th width="3%"> {{translate 'FORM.LABELS.ACCOUNT_NUMBER'}} </th>
+            <th width="10%"> {{translate 'FORM.LABELS.LABEL'}} </th>
+            <th width="5%"> {{translate 'FORM.LABELS.BUDGET'}} {{ fiscalYearNumber }} </th>
+            <th width="5%"> {{translate 'REPORT.BUDGET_REPORT.REALIZATION'}} {{ fiscalYearNumber }} </th>
+            {{#each reportColumn as | col |}}
+              <th width="5%">
+                {{translate 'REPORT.BUDGET_REPORT.REALIZATION'}} {{ col.number }}
+              </th>  
+            {{/each}}
+            <th width="5%"> {{translate 'REPORT.BUDGET_REPORT.VARIATION_IN_AMOUNT'}} {{ fiscalYearNumber }} - {{ lastFiscalYearNumber }} </th>
+            <th width="10%"> {{translate 'REPORT.BUDGET_REPORT.PERCENTAGE_VARIATION_COMPARED'}} {{ fiscalYearNumber }} </th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {{#each rowsExpense as | row |}}
+            {{#if isTitle}}
+            <tr style="font-weight: bold; font-style: italic">
+            {{else}}
+            <tr>
+            {{/if}}
+              <td> {{ row.number }} </td>
+              <td> {{ row.label }} </td>
+              <td class="text-right"> {{currency row.budget ../currencyId}} </td>
+              <td class="text-right"> {{currency row.realisation ../currencyId}} </td>
+              {{#each row.previousReport as | cl |}}
+                <td class="text-right">
+                  {{currency cl.realisation ../../currencyId}}
+                </td>
+              {{/each}}
+              <td class="text-right"> {{currency row.variation ../currencyId}} </td>
+              <td class="text-right"> {{percentage row.completion}} </td>     
+            </tr>
+          {{/each}}
+          <tr style="background-color: #E1E1E1; font-weidth: bold; font-size: 10pt;">
+            <th colspan="2"> {{translate 'BUDGET.EXPENSES_TOTAL'}} </th>
+            <th class="text-right">{{currency totalBudgetExpense currencyId}}</th>
+            <th class="text-right">{{currency totalRealisationExpense currencyId}}</th>
+            {{#each reportFootColumExpense as | cl |}}
+              <th class="text-right">
+                {{currency cl.realisation ../currencyId}}
+              </th>
+            {{/each}}
+
+            <th class="text-right">{{currency totalVariationExpense currencyId}}</th>
+            <th class="text-right">{{percentage totalCompletionExpense}}</th>
+          </tr>          
+        </tbody>
+      </table>
+
+
+
+      {{!-- <table class="table table-striped table-condensed table-report table-bordered">
```

Comment:
What is this?

Reply:
It's a saving error, I had deleted but unfortunately the saving of the file had not taken effect

Reply:
This obsolete table still needs to be removed.